modules/programs/discocss.nix

```
@@ -0,0 +1,56 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.discocss;
+in {
+  meta.maintainers = with maintainers; [ kranzes ];
+
+  options = {
+    programs.discocss = {
+      enable = mkEnableOption
+        "discocss, a tiny Discord CSS injector for Linux and MacOS";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.discocss;
+      };
+
+      discord = mkOption {
+        type = types.package;
+        default = pkgs.discord;
+        description = "The Discord package for discocss to use";
+      };
+
+      discordAlias = mkOption {
+        type = types.bool;
+        default = true;
+        description = "Whether to alias discocss to discord.";
+      };
+
+      css = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        description = "The custom CSS for discocss to use.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    assertions = [{
+      assertion = cfg.discordAlias
+        -> !(builtins.any (p: p.name == cfg.discord.name) config.home.packages);
```

I think it is sufficient to have

```suggestion
        -> !(any (p: p.name == cfg.discord.name) config.home.packages);
```

---

`any` isn't in scope, is it?

---

oh it's coming from nixpkgs' lib.